software/controller/lib/debugvars/vars_base.h

```diff
@@ -18,6 +18,8 @@ limitations under the License.
 #include <cstddef>
 #include <cstdint>
 
+#include "nvparams.h"
```

Comment:
This file has become a bit of a dumping ground as well -- maybe time to split this off?

Reply:
absolutely